frontend/packages/helm-plugin/src/components/list-page/HelmReleaseListPage.tsx

```diff
@@ -23,7 +23,7 @@ const PageContents: React.FC<HelmReleaseListPageProps> = (props) => {
   return namespace ? (
     <div>
       <PageHeading title={t('helm-plugin~Helm Releases')} />
-      <HelmReleaseList namespace={namespace} />
+      <HelmReleaseList match={props.match} />
```

Comment:
```suggestion
      <HelmReleaseList namespace={props.match.params.ns} />
```

Reply:
HelmReleaseList is also called in HelmTabbedPage.tsx 

There we get match as prop. So I am passing match as prop in HelmReleaseListPage.tsx